apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingWithdrawns/StakingWithdrawns.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { FC, useCallback, useEffect, useMemo, useState } from 'react';
+
+import { nanoid } from 'nanoid';
+import { useTranslation } from 'react-i18next';
+
+import {
+  ErrorBadge,
+  ErrorLevel,
+  NotificationType,
+  OrderDirection,
+  OrderOptions,
+  Pagination,
+  Select,
+  Table,
+} from '@sovryn/ui';
+
+import { ExportCSV } from '../../../../2_molecules/ExportCSV/ExportCSV';
+import {
+  DEFAULT_HISTORY_FRAME_PAGE_SIZE,
+  EXPORT_RECORD_LIMIT,
+} from '../../../../../constants/general';
+import { useNotificationContext } from '../../../../../contexts/NotificationContext';
+import { useAccount } from '../../../../../hooks/useAccount';
+import { useMaintenance } from '../../../../../hooks/useMaintenance';
+import { translations } from '../../../../../locales/i18n';
+import { rskClient } from '../../../../../utils/clients';
+import {
+  useGetV2StakingWithdrawnsLazyQuery,
+  V2StakingWithdrawn_OrderBy,
+} from '../../../../../utils/graphql/rsk/generated';
+import { dateFormat } from '../../../../../utils/helpers';
+import { stakingHistoryOptions } from '../../StakingHistoryFrame.constants';
+import { StakingHistoryProps } from '../../StakingHistoryFrame.type';
+import { COLUMNS_CONFIG } from './StakingWithdrawns.constants';
+import { generateRowTitle } from './StakingWithdrawns.utils';
+import { useGetStakingWithdrawns } from './hooks/useGetStakingWithdrawns';
+
+const pageSize = DEFAULT_HISTORY_FRAME_PAGE_SIZE;
+
+export const StakingWithdrawns: FC<StakingHistoryProps> = ({
```

Comment:
```suggestion
export const StakingWithdraws: FC<StakingHistoryProps> = ({
```
Use `StakingWithdraws` or `StakingWithdrawals`.